### PR TITLE
Key value pairs

### PR DESCRIPTION
**Pull request by simPod**
And now this is followup to https://github.com/Vertamedia/clickhouse-grafana/pull/66

### CODE REVIEW

**Comment by simPod** [comment]
I'll have to refactor tests first, I'm unable to run them.

**Comment by simPod** [comment]
@hagen1778 now that tests work for me flawlessly cuz of #107 I added a test. Is it k now?

File: README.md [hunk 1]
```diff
@@ -279,6 +279,37 @@ ORDER BY t ASC
...
+If you add a template variable of the type `Query`, you can write a ClickHouse query that can
+return things like measurement names, key names or key values that are shown as a dropdown select box.
+ 
+For example, you can have a variable that contains all values for the `hostname` column in a table if you specify a query like this in the templating variable *Query* setting.
+ 
+```sql
+SELECT hostname FROM host
+```
+ 
+To use time range dependent macros like `$timeFilterColumn($column)` in your query the refresh mode of the template variable needs to be set to *On Time Range Change*.
```
Comment (hagen1778) [inline README.md]: What is the `timeFilterColumn` macros? I can't find it's definition

Comment (simPod) [reply README.md]: It's from this PR https://github.com/Vertamedia/clickhouse-grafana/pull/94 Originally it was `timeFilterColumn` and we changed it to `timeFilterByColumn`. Fixing. Good catch

**Pull request merged**